src/daemon/org.projectatomic.rpmostree1.xml

```diff
@@ -345,6 +345,7 @@
          "override-replace-packages" (type 'as')
          "override-replace-local-packages" (type 'ah')
          "custom-origin" (type '(ss)')
+         "treefile" (type 's')
```

Comment:
Do you think it make sense in the future to add a new `UpdateDeploymentFromTreefile` that *only* accepts a treefile?  And it'd take a subset of modifiers/options.